lib/wallet/interface.ts

```diff
@@ -89,4 +95,10 @@ export interface IWallet {
   ): Promise<string>;
 
   signTx(tx: Transaction, scripts?: ScriptOption[]): Promise<Transaction>;
+
+  performCommitReveal(
+    scriptBuilder: ScriptBuilder,
+    revealPriorityFee: IGeneratorSettingsObject["priorityFee"],
+    extraOutputs?: IPaymentOutput[],
+  ): AsyncGenerator<"commiting" | "revealing", void, unknown>;
```

Comment:
What is `void` and `unknown` in this output?
Would it better to have a object like:
```
{
  status: "commited" | "revealed",
  commitTxId?: string,
  revealTxId: string,
}
```